module_renamer/commands/rename_imports.py

```diff
@@ -0,0 +1,91 @@
+from concurrent import futures
+
+import pasta
+import six
+from click._unicodefun import click
+from pasta.augment import rename
+from tqdm import tqdm
+
+from module_renamer.commands.utils import walk_on_py_files
+
+
+def rename_modules(project_path, path_to_moved_imports_file):
+    for path in project_path:
+        execute_rename(path, path_to_moved_imports_file)
+
+
+def execute_rename(project_path, path_to_moved_imports_file):
+    """
+    Main loop that interacts over all python files from the project and delegate
+    to an executor to parse each file
+
+    :param str project_path:
+        Path to the project that is going to be parsed.
+
+    :param str path_to_moved_imports_file:
+        Path of the file with the list of changed imports.
+    """
+    list_of_py_files = [i for i in walk_on_py_files(project_path)]
```

Comment:
No need to change, but you can simply write: `list(walk_on_py_files(project_path))` as well 😉

Reply:
Thanks for the tip ;) 
I wasn't aware that list() could consume an iterable